### ISSUE TITLE
Unlocking quick travel points no longer required

### ISSUE THREAD

**Issue by pjmagee** [BosslandGmbH/BuddyWing#37]
This happens automatically, so maybe if there was a way to enable or disable / ignore anything like this in profiles. It could speed up and make things run a bit smoother.

**Comment by aevitas** [comment]
Could you give me an example of what part of the profiles handle this?

**Comment by pjmagee** [comment]
``` javascript
<Interact Name="Senate Commercial District Cantina" Radius="20" UseRange="5" IgnoreLOS="true" X="-122.6072" Y="141.5989" Z="-445.1695" />   <!-- Bind point: Senate Commercial District Cantina -->
```

I presume BuddyWing would need to know what type of object it was clicking in order to just ignore this request from the profile.

**Comment by aevitas** [comment]
If I recall correctly, they're all simply `TorPlaceable` objects - we don't know whether that's a quest terminal, a travel point, or a quest object that we need to loot. Removing these lines from the profiles would make more sense than handling it in the bot's code I think?

**Comment by pjmagee** [comment]
Yes, I think  you are right. Deleting anything in a profile that clicks a bind point is probably the right thing to do.

**Comment by CryoGenesisX** [comment]
The best course is to remove these from the profiles.
To much variables for the bot to block this.

**Comment by pjmagee** [comment]
So do we close this issue off? Simply load the XML profiles into a mini C# program that goes through them and finds any `<!-- bind point` and deletes the node? 

I was also thinking about changing the UseTransport to replace it with a MoveTo and then a Message pop up telling the user to "Click" the transport and use it. Since all these leveling profiles are unusable in their current state due to the crashing bugs for travelling or using lifts.

**Comment by CryoGenesisX** [comment]
I changed the code from Kick to better navigate the bot to that bindpoint.
So its not only the bindpoint but also the moveto infront of it.
So removing those 2 lines in my code is easy, or only the the bindpoint line in kicks.

For elevator issue, Aevitas just needs to fix it :)

**Comment by pjmagee** [comment]
Same for the Taxi

**Comment by derFalke** [comment]
the problem is that the scripts, which are available, have to be reworked thats all. No bug in the engine.
